src/main/bash/NodeLimit/limit.sh

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env bash
+#
+#Script for creating a cpu limited cgroup based on information retrieved as a REST resource
+
+no_quotes_regex='"\K\w+(?=")' ##Regular expression to match a word between quotes, ignoring the quotes
+
+curl -s http://localhost:4567/limit | grep -Po '"\w+":"\w+"' > limitFile ##Do the GET request and save the data to a file
```

Comment:
That will only read the limits once, we may have dynamic events that affect the limits, in such case the script should read from the REST resource again.

I would say make it a part of a loop (new one of course).

Another thing, make read the URL as a parameter of the script.